dictionary.json

```diff
@@ -10175,7 +10363,7 @@
     "toaq": "mateoq",
     "type": "predicate",
     "english": "▯ is a polar yes-no question.",
-    "gloss": "",
+    "gloss": "yes-no.question",
```

Comment:
`-` is used in glosses for indicating segmentable morphemes, this should maybe be `yes.no.question`?